interfaces/cython/setup.cfg.in

```diff
@@ -49,16 +49,14 @@ packages =
     cantera.data
     cantera.test
     cantera.test.data
-    cantera.examples
 
 [options.package_data]
 cantera.data = *.*, */*.*
 cantera.test.data = *.*, */*.*
-cantera.examples = */*.*
 # The module extension needs to be here since we don't want setuptools to
 # compile the extension, so there are no `source` files in
 # the setup.py extension.
```

Comment:
This is unrelated to your changes, but I wanted to add a small clarification to this comment as I was reading the file :smile:
```suggestion
# the setup.py ``extension`` and we have to treat the module as package data.
```